lib/preprocess.js

```diff
@@ -103,3 +121,21 @@ const assertValidMeasurement = measurement => {
   assert(typeof measurement.inet_group === 'string', 'valid inet group required')
   assert(typeof measurement.finished_at === 'string', 'field `finished_at` must be set to a string')
 }
+
+/**
+ * @param {import('./typings').Measurement} measurement
+ * @return {import('./typings').RetrievalResult}
+ */
+export const calculateRetrievalResult = measurement => {
```

Comment:
`calculate` is a misleading name, as nothing is being calculated. What about `getRetrievalResult` or `mapRetrievalResult`?